quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/integration/web/QuarkusRequestFilter.java

```diff
@@ -68,89 +67,43 @@ public void handle(RoutingContext context) {
         }
         // our code should always be run as blocking until we don't provide a better support for running non-blocking code
         // in the event loop
-        context.vertx().executeBlocking(createBlockingHandler(context), false, EMPTY_RESULT);
+        executor.execute(createBlockingHandler(context));
```

Comment:
I see that this now changes to the executor thread pool. What were the reasons for that, and how would I configure the thread pool boundaries? 

If the execution throws an exception, where would it be logged? The old code just swallowed it, what would make analysis of problems difficult.

Reply:
The executor is the same. We are basically using it directly instead of dispatching using `RoutingContext`. This is basically a cosmetic change as it makes the code much simpler, with the possibility of managing the request lifecycle a lot better as we can deal with completable|futures to add very nice non-functional aspects. For instance, better exception handling.

I'm not doing it now as I want to just rely on the Vert.X/Quarkus/Resteasy error handing for now.

Reply:
I'd be happy if you could verify with a manual test how those exceptions in the future are handled in the Vert.X/Quarkus/Resteasy error handing. I'd hope they would be logged.

Reply:
We have a test covering this. The FlowTest expects a response error during commit. Or do you mean something else?

Reply:
I've tested what happens when an exception would happen after `context.next()`. I now see that the executor pool logs that information: 

```
2022-10-31 10:54:58,128 ERROR [org.jboss.threads.errors] (executor-thread-1) Thread Thread[executor-thread-1,5,main] threw an uncaught exception: java.lang.RuntimeException: java.lang.RuntimeException: oops
	at org.keycloak.quarkus.runtime.integration.web.QuarkusRequestFilter.lambda$createBlockingHandler$0(QuarkusRequestFilter.java:88)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:564)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.RuntimeException: oops
	at org.keycloak.quarkus.runtime.integration.web.QuarkusRequestFilter.lambda$createBlockingHandler$0(QuarkusRequestFilter.java:84)
	... 7 more
```

So this is resolved for me.